pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -216,9 +228,23 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 		o.CurrentReports = append(o.CurrentReports, report)
 		allExecutionIDs = append(allExecutionIDs, weid.WorkflowExecutionId)
 
+		if seenWorkflows[weid.WorkflowId] >= (r.config.F + 1) {
```

Comment:
This is incorrect since it will only update LastSeenAt if the id was part of the query (q.Ids); we want to update this for all registered workflows regardless of whether they were in the query or not (of course assuming F+1)

Reply:
So you probably want to loop over seenWorkflows